lang/cs/Org.Apache.REEF.IMRU/OnREEF/Driver/ActiveContextManager.cs

```diff
@@ -71,7 +71,7 @@ internal int NumberOfMissingContexts
         /// </summary>
         /// <param name="activeContextObserver"></param>
         /// <returns></returns>
-        public IDisposable Subscribe(IObserver<IDictionary<string, IActiveContext>> activeContextObserver)
+        public IDisposable Subscribe(IObserver<IAllContextsReceived> activeContextObserver)
```

Comment:
For the few *Managers, EvaluatorManager needs two EvaluatorSpecification from the same class, Tang doesn't support. ActiveContextManager needs totalContexts that is got from IMRUDriver dataSet.Count and IPartitionedInputDataSet is injected in IMRUDriver, unless we inject IPartitionedInputDataSet in the ActiveContextManager constructor, which seems unnecessary. Similar things are for the parameters used in TaskManager. To make it consistent, creating their instances in IMRUDriver is clean.